web-services/model/src/main/java/datawave/webservice/query/model/ModelKeyParser.java

```diff
@@ -5,39 +5,37 @@
 import org.apache.accumulo.core.data.Key;
 import org.apache.accumulo.core.data.Mutation;
 import org.apache.accumulo.core.data.Value;
-import org.apache.accumulo.core.security.Authorizations;
 import org.apache.accumulo.core.security.ColumnVisibility;
 import org.apache.commons.lang.StringUtils;
 import org.apache.log4j.Logger;
 
-import java.util.Set;
+import java.util.Arrays;
 
 public class ModelKeyParser {
     
     public static final String NULL_BYTE = "\0";
     public static final Value NULL_VALUE = new Value(new byte[0]);
-    private static Logger log = Logger.getLogger(ModelKeyParser.class);
     
-    public static FieldMapping parseKey(Key key, Set<Authorizations> auths) {
-        FieldMapping mapping = new FieldMapping();
+    private static final Logger log = Logger.getLogger(ModelKeyParser.class);
+    
+    public static FieldMapping parseKey(Key key) {
```

Comment:
I haven't read the entire PR yet, so the answer to this may be elsewhere, but I'm curious why we have dropped the columnVisibility from the `FieldMapping` produced by this method.

From what I can tell `auths` was unused so it might sense to get rid of them here, but I wonder if there are subclasses elsewhere that may depend on this method signature.

Reply:
@drewfarris I did not have any issues with subclasses needing the original `parseKey(Key key, Set<Authorizations> auths)` method signature, but I can add it back as an overloaded method and call `parseKey(Key key)` within it.

Reply:
It appears that the visibility was added back in.  I am checking to see if there are any other dependencies on parseKey(key, auths)

Reply:
confirmed that the only place parseKey was called is in the ModelBean.  So dropping the arg is ok